lldb-eval/BUILD

```diff
@@ -0,0 +1,92 @@
+load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
+
+package(
+    default_visibility = ["//visibility:public"],
+)
+
+COPTS = select({
+    "@bazel_tools//src/conditions:windows": [
+        # Mark the LLVM includes as external
+        "/external:I external/llvm_project/include",
+    ],
+    "//conditions:default": [],
+})
+
+cc_library(
+    name = "lldb-eval",
+    srcs = [
+        "api.cc",
+        "ast.cc",
+        "eval.cc",
+        "expression_context.cc",
+        "parser.cc",
+        "pointer.cc",
+        "scalar.cc",
+        "value.cc",
+    ],
+    hdrs = [
+        "api.h",
+        "ast.h",
+        "defines.h",
+        "eval.h",
+        "expression_context.h",
+        "parser.h",
+        "pointer.h",
+        "scalar.h",
+        "value.h",
+    ],
+    copts = COPTS,
+    deps = [
+        "@llvm_project//:clang-basic",
+        "@llvm_project//:clang-lex",
+        "@llvm_project//:lldb-api",
+        "@llvm_project//:llvm-support",
+    ],
+)
+
+cc_test(
+    name = "eval_test",
+    srcs = ["eval_test.cc"],
+    copts = COPTS,
+    tags = [
+        # On Linux lldb-server behaves funny in a sandbox ¯\_(ツ)_/¯. This is
+        # not necessary on Windows, but "tags" attribute is not configurable
+        # with select -- https://github.com/bazelbuild/bazel/issues/2971.
+        "no-sandbox",
+    ],
+    deps = [
+        ":lldb-eval",
+        ":runner",
+        "@com_google_googletest//:gtest",
+        "@com_google_googletest//:gtest_main",
+        "@llvm_project//:lldb-api",
+    ],
+)
+
+cc_test(
```

Comment:
Maybe it would be a good idea to keep the testcases in a separate directory/separate directories?

Reply:
With Bazel and gtest it's common to have `lib.cc` and `lib_test.cc` next to each other. IMO it's a good practice (at least for now), it keeps the test local and easy to discover.